internal/builder/lifecycle_test.go

```diff
@@ -46,12 +44,9 @@ func testLifecycle(t *testing.T, when spec.G, it spec.S) {
 		})
 
 		when("there is no descriptor file", func() {
-			it("assumes Platform API version 0.1", func() {
-				lifecycle, err := builder.NewLifecycle(&fakeEmptyBlob{})
-				h.AssertNil(t, err)
-				h.AssertEq(t, lifecycle.Descriptor().Info.Version.String(), "0.3.0")
-				h.AssertEq(t, lifecycle.Descriptor().API.PlatformVersion.String(), "0.1")
-				h.AssertEq(t, lifecycle.Descriptor().API.BuildpackVersion.String(), "0.1")
+			it("throws an error ", func() {
+				_, err := builder.NewLifecycle(blob.NewBlob(filepath.Join("testdata", "lifecycle-bad")))
```

Comment:
Is there a reason why the filesystem was used instead of the empty blob? We are moving towards reducing the overhead of maintaining fixtures in certain areas where the same effect could be achieved via inversion of control. The method `builder.NewLifecycle` shouldn't care if it's a file system based blob or an in-memory one.